karate-core/src/main/java/com/intuit/karate/ui/StepPanel2.java

```diff
@@ -103,10 +110,10 @@ public void initStyles() {
         StepResult sr = unit.result.getStepResult(index);
         if (sr == null) {
             runButton.setStyle("");
-        } else if (sr.getResult().isFailed()) {
-            runButton.setStyle(STYLE_FAIL);
+        } else if (sr.getResult().getStatus().equals("passed")) {
```

Comment:
any reason to change this ?

Reply:
The intention is to show green on the pass result and red for remaining cases